frontend/src/components/Footer/Footer.module.css

```
@@ -17,30 +17,36 @@
   transform: translateX(-6px);
 }
 
-.copyright {
-  margin-bottom: 1.25rem;
-}
-
 .social {
   display: flex;
+  align-items: center;
 }
 
 .social a {
-  width: 2.75rem;
-  height: 2.75rem;
-  color: var(--white);
-  margin-right: 1rem;
   display: flex;
   justify-content: center;
   align-items: center;
-  border-radius: 100%;
-  border: 1px solid #fff;
+  padding-left: 1rem;
+  padding-right: 1rem;
```

다른 css 파일의 경우 padding: y x의 형식으로 작성했네요. 통일하면 좋을 것 같습니다.

```suggestion
  padding: 0 1rem;
```

---

저 또한 우디의 의견에 공감합니다.